src/Symfony/Component/VarDumper/Tests/Caster/ReflectionCasterTest.php

```diff
@@ -292,6 +313,34 @@ public function testReflectionUnionType()
         );
     }
 
+    /**
+     * @requires PHP 8.1
+     */
+    public function testReflectionIntersectionType()
+    {
+        $var = (new \ReflectionProperty(ReflectionIntersectionTypeFixture::class, 'a'))->getType();
+        $this->assertDumpMatchesFormat(
+            <<<'EOTXT'
+ReflectionIntersectionType {
+  allowsNull: false
+  types: array:2 [
+    0 => ReflectionNamedType {
+      name: "Traversable"
+      allowsNull: false
+      isBuiltin: false
+    }
+    1 => ReflectionNamedType {
+      name: "Countable"
+      allowsNull: false
+      isBuiltin: false
+    }
+  ]
```

Comment:
I've decided to match the generated output for `ReflectionUnionType` here. This is a bit verbose though because the sub-types of an intersection are guaranteed to be a `ReflectionNamedType` with `$allowsNull === false` and `$isBuiltin === false`.

Reply:
Actually, nullable intersection types are possible, see https://3v4l.org/aURFN/rfc#vgit.master

Reply:
See https://bugs.php.net/81268

Reply:
Oh well. It's not like I haven't told them. 🤷🏻  https://github.com/php/php-src/pull/6799#issuecomment-805175050

Reply:
The loophole was closed. php/php-src#7254